src/index.js

```diff
@@ -1,7 +1,33 @@
-import { sanitizeInputCommand } from './utils/helper.js';
+import { sanitizeInputCommand , readAndParseTomlConfig} from './utils/helper.js';
 import fs from 'fs';
 let args = process.argv.slice(2);
 
+// Check for config flag
+const configFlagIndex = args.findIndex(arg => arg === '-c' || arg === '--config');
+
+if (configFlagIndex !== -1 && args[configFlagIndex + 1]) {
```

Comment:
Could you please add a small comment explaining what does this condition do?

Reply:
The "Check for config flag" comment was fine. With the latest commit you added the explanation to line 6 instead of line 8 for line 9. :) @mnajibi

Reply:
Solved in commit [09791](https://github.com/sdthaker/Node-TILify/pull/13/commits/09791bfc36a7c5b601649075c18c845a87c17d7c)